src/main/java/com/flightstats/hub/metrics/DataDogHandler.java

```diff
@@ -0,0 +1,65 @@
+package com.flightstats.hub.metrics;
+
```

Comment:
This was formerly part of metricsService (now ~ statsdReporter) I broke it out and refactored very little, it is not 1:1 copy paste but further refactoring, I felt was too much on this already large PR